apps/wallet/src/background/keyring/index.ts

```diff
@@ -308,6 +313,18 @@ export class Keyring {
                         id
                     )
                 );
+            } else if (
+                isKeyringPayload(payload, 'importPrivateKey') &&
+                payload.args
+            ) {
+                const imported = await this.importAccountKeypair(
+                    payload.args.keyPair,
+                    payload.args.password
+                );
+                if (!imported) {
+                    throw new Error('Duplicate account not imported');
```

Comment:
I noticed a small issue here: 

1. Create a new account
2. Export / Copy private key
3. Import using same private key

The first attempt shows "Account imported successfully" message, the second attempt shows the "Duplicate account not imported message"

Reply:
Yeah thats known, I mentioned during the bug bash meeting. But thanks for reminding me about it. I will add fix to check accounts that are derived from mnemonic as well. Although still it wont be perfect since the imported account can be a derived one but not yet 'created' in the wallet.

I plan to add it as a separate PR though. here is the jira ticket https://mysten.atlassian.net/browse/APPS-534